stories/Button.stories.svelte

```diff
@@ -0,0 +1,59 @@
+<script>
+  import { Meta, Template, Story } from "@storybook/addon-svelte-csf";
+  import { Button } from "../components/mdc";
+</script>
+
+<Meta
+  title="Atoms/Button"
+  component={Button}
+/>
+
+<Template let:args>
+  <Button {...args} on:click={args.onClick}>{args.label}</Button>
+</Template>
+
+<Story
+  raised
+  name="Primary"
+  args={{
+    raised: true,
+    label: "Button",
+  }}
+/>
+
+<Story
+  name="Secondary"
+  args={{
+    label: "Button",
+    outlined: true,
+    raised: true,
+  }}
+/>
+
+<Story
+  name="Disabled"
```

Comment:
Do you have to write this all up or does storybook generate it from our components? Some of the stories like 'Disabled' might not be essential as they can be set by the prop on the other stories.